src/Cake.Common/Tools/NUnit/NUnitRunner.cs

```diff
@@ -124,6 +124,16 @@ private ProcessArgumentBuilder GetArguments(IEnumerable<FilePath> assemblyPaths,
                 builder.Append("/trace:" + settings.Trace);
             }
 
+            if (settings.OutputFile != null)
+            {
+                builder.AppendQuoted(string.Format(CultureInfo.InvariantCulture, "/output:{0}", settings.OutputFile.MakeAbsolute(_environment).FullPath));
```

Comment:
According to the [NUnit-Console Command Line Options](http://www.nunit.org/index.php?p=consoleCommandLine&r=2.6.3) page the parameter name should be `/out:`.

Is `/output` an alias?